modules/vscode/hm.nix

```
@@ -40,44 +45,10 @@ in
   };
 
   config = lib.mkIf (config.stylix.enable && cfg.enable) {
-    programs.vscode.profiles = lib.mkMerge (
-      map (profileName: {
-        ${profileName} = {
-          extensions = [ themeExtension ];
-          userSettings = {
-            "workbench.colorTheme" = "Stylix";
-            "editor.fontFamily" = monospace.name;
-            "editor.inlayHints.fontFamily" = monospace.name;
-            "editor.inlineSuggest.fontFamily" = monospace.name;
-            "scm.inputFontFamily" = monospace.name;
-            "debug.console.fontFamily" = monospace.name;
-            "markdown.preview.fontFamily" = sansSerif.name;
-            "chat.editor.fontFamily" = monospace.name;
-
-            # 4/3 factor used for pt to px;
-            "editor.fontSize" = sizes.terminal * 4.0 / 3.0;
-            "debug.console.fontSize" = sizes.terminal * 4.0 / 3.0;
-            "markdown.preview.fontSize" = sizes.terminal * 4.0 / 3.0;
-            "terminal.integrated.fontSize" = sizes.terminal * 4.0 / 3.0;
-            "chat.editor.fontSize" = sizes.terminal * 4.0 / 3.0;
+    programs.vscode.profiles = lib.genAttrs cfg.profileNames (_name: profile);
```

What about having no name:

```suggestion
    programs.vscode.profiles = lib.genAttrs cfg.profileNames (_: profile);
```

However, I am fine keeping the name.